test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -819,6 +820,74 @@ ava('._prependLegCollectionWithRouteAmendment() adds LegModels for each routeStr
     t.true(fms.legCollection[1].routeString === routeAmmendmentMock[1]);
 });
 
+ava('._regenerateSidLeg() creates a new SID leg of identical route string for the currently assigned departure runway', (t) => {
```

Comment:
I'm not seeing any instances of multiple matchers, but I wasn't inclined to avoid it since I didn't know. I'll keep that in mind going forward.